### ISSUE TITLE
HIFI Berry MiniAmp not showing up

### ISSUE THREAD

**Issue by gburkell** [mikebrady/shairport-sync#1078]
Hi Guys,
I am building a system with a Pi Zero W and a [HIFIBerry MiniAmp](https://www.hifiberry.com/shop/boards/miniamp/). I got everything working great through the headphone output, but when I put the Hifiberry minimap hat on, it doesn't show up in the Alsamixer so I can't change to it.  

Is this usual? Any suggestions on how to trouble shoot this issue?

Thanks for the help!

**Comment by mikebrady** [comment]
Thanks for the post. If you do:
```
$ shairport-sync -h
```
then at the very end, you'll get a list of the `alsa` output devices that Shairport Sync can "see". For example, this is from a Raspberry Pi 3B plus, fully up to date, with an IQaudIO Pi-DigiAMP+:
```
pi@kitchen:~ shairport-sync -h
Usage: shairport-sync [options...]
  or:  shairport-sync [options...] -- [audio output-specific options]

Options:
    -h, --help              show this help.
    -V, --version           show version information.

...
[snip]
...

Settings and options for the audio backend "alsa":
    -d output-device    set the output device, default is "default".
    -c mixer-control    set the mixer control name, default is to use no mixer.
    -m mixer-device     set the mixer device, default is the output device.
    -i mixer-index      set the mixer index, default is 0.
    hardware output devices:
      "hw:Headphones"
      "hw:IQaudIODAC"
pi@kitchen:~ $ 
```
The "hw:Headphones" device is the built-in device, so in this case I would set the output device to `"hw:IQaudIODAC"` to access the IQaudIO amplifier.

If Shairport Sync can't see the device, then it sounds like some kind of configuration issue with the Pi.

**Comment by gburkell** [comment]
Great, I'm just at work and will try this tonight ;-)

**Comment by gburkell** [comment]
OK,
So you got me on the right direction, in the end it was user error, for some reason I assumed that it would be recognized when I placed it on. It wasn't.  For future people using this, I went to the [Hifiberry site and found this install guide](https://www.hifiberry.com/docs/data-sheets/datasheet-miniamp/).  Then rebooted and Shairport saw it, then using the Alsamixer I figured out the corresponding number and changed that in the .conf file and altered the number there. Rebooted and it works.

Thanks Mike!
